index.html

```diff
@@ -249,6 +249,8 @@ <h3 id="selectHardwareAcceleration">Hardware Acceleration</h3>
 															<span>Vitis AI (Preview)</span></div>
 														<div class="col-lg-3 col-md-3 r-option version" role="option" tabindex="-1" aria-selected="false" id="XNNPACK">
 															<span>XNNPACK (Preview)</span></div>
+														<div class="col-lg-3 col-md-3 r-option version" role="option" tabindex="-1" aria-selected="false" id="Azure">
+															<span>Azure (Preview)</span></div>
```

Comment:
This should be alphabetical in the "preview" portion of the list. So it should be after "ArmNN"

Reply:
Moved.